app/controllers/likes_controller.rb

```diff
@@ -1,4 +1,6 @@
 class LikesController < ApplicationController
+  load_and_authorize_resource
```

Comment:
- [ ] [OPTIONAL]
Similarly, we only intend to allow authorization of resources on _Post_, and hence it's redundant to include `load_and_authorize_resource` in the _Like_ controller.